source/sass/buyers-guide/views/product.scss

```diff
@@ -4,6 +4,26 @@
 
 .product-detail {
   margin-top: -88px;
+
+  @media (min-width: $bp-md) {
+    .intro {
+      position: relative;
+
+      .h1-heading {
+        margin-right: 8rem;
```

Comment:
Hm, could we not achieve sufficient margin with bootstrap?

Reply:
Nope, Bootstrap margin classes only go up to `5` e.g,. `mb-5` which adds 3rem to margin bottom and this is the max margin we can get from Bootstrap

Reply:
Should that element just be in its own column then?

Reply:
that's probably worth doing, since that suggests it's not on the right bootstrap grid if we need that much margin.

Reply:
I'm not sure how we can achieve our mobile and desktop design solely by using bootstrap.

(pay attention to the order of "copy link")
<img width="900" alt="image" src="https://user-images.githubusercontent.com/2896608/47305849-9fc96780-d5df-11e8-96c4-ee18a5297ea8.png">

I was thinking to rely on flexbox's `order` but since the 3 elements title, copy link, description aren't even on the same row on desktop, flexbox's `order` won't help. Suggestions?

Reply:
@alanmoo @Pomax ^

Reply:
I would think you should be able to use markup like this:

```
<div class="row">
<div class="title col-9">title</div>
<div class="copy col-3">copy link</div>
<div class="blurb col-12">paragraph</div>
</div>
```

And then add `order` to the unique classnames using CSS and some bootstrap media queries. (Those aren't actually the column classes you'd need, they're just for example.)

Reply:
awesome! i like this clean approach